### PR TITLE
NSW Rural Fire Service platform

### PR DESCRIPTION
**Pull request by exxamalte**
**Description:**
Description of how to integrate the new NSW Rural Fire Service platform.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16802

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Review by DubhAd — DISMISSED** [review]
👍

File: source/_components/geo_location.nsw_rural_fire_service_feed.markdown [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+description: "Instructions on how to integrate the NSW Rural Fire Service Incidents feed into Home Assistant."
+date: 2018-09-23 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nsw-rural-fire-service.png
+ha_category: Geo Location
+ha_iot_class: "Cloud Polling"
+ha_release: "0.80"
```
Comment (MartinHjelmare) [inline source/_components/geo_location.nsw_rural_fire_service_feed.markdown]: 0.81

File: source/_components/geo_location.nsw_rural_fire_service_feed.markdown [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# Example configuration.yaml entry
+geo_location:
+  - platform: nsw_rural_fire_service_feed
+```
+
+{% configuration %}
+radius:
+  description: The distance in kilometers around Home Assistant's coordinates in which incidents are included.
+  required: false
+  type: string
```
Comment (MartinHjelmare) [inline source/_components/geo_location.nsw_rural_fire_service_feed.markdown]: Do we have float as option?

File: source/_components/geo_location.nsw_rural_fire_service_feed.markdown (hunk 1 above)
Comment (exxamalte) [reply source/_components/geo_location.nsw_rural_fire_service_feed.markdown]: OK

File: source/_components/geo_location.nsw_rural_fire_service_feed.markdown (hunk 2 above)
Comment (exxamalte) [reply source/_components/geo_location.nsw_rural_fire_service_feed.markdown]: Yes, it is actually a float

**Review by MartinHjelmare — APPROVED** [review]
Nice! Can be merged when build passes.

**Comment by Captain-Glen** [comment]
Hi,

This plugin is great!!!

But I wonder if you could please update the documentation on how to use it in an automation.  Specifically how to create a trigger if any fire is within a certain distance and then send an email using the notify: -platform smtp plugin.

I suspect your code allows this.  But I haven't been able to figure out how to use it in an automation.

**Comment by exxamalte** [comment]
Thanks @Captain-Glen - the automation part is covered in https://github.com/home-assistant/home-assistant.io/pull/7041 where I describe how to use the new `geo_location` trigger. Not sure when that will be online.

**Comment by Captain-Glen** [comment]
Thanks Malte,

I got it working.  It sends me an email when a fire is within 100 km. And
when it disappears i.e. gets extinguished.

Is a great add-on for home assistant.  It's really useful now that it's
bush fire season.

It would be good for tech savvy farmers with smart sprinklers etc.

Thanks again,
Glen.

**Pull request merged**